File: atcoder/abc/abc091/c1.rb
N=gets.to_i
red = []
blue = []
ans = 0
N.times do
  red << gets.split.map(&:to_i)
end
N.times do
  blue << gets.split.map(&:to_i)
end

blue.sort_by!{|pos| pos[0]}
red.sort_by!{|pos| -pos[1]}
ans = 0
blue.each do |b|
  red.each do |a|
    if b[0] > a[0] && b[1] > a[1]
      red.delete(a)
      ans += 1
      break
    end
  end
end
puts ans

File: atcoder/abc/abc042/c.1.py
N,K=map(int, input().split())
D=list(map(int, input().split()))
use=[]
for i in range(10):
    if i not in D:
        use.append(i)
ans = ''

for n in list(map(int, list(str(N)))):
    if n > use[-1]:
        ans+=str(use[0])
    else:
        for i in use:
            if i >= n:
                ans+=str(i)
                break

if str(N)[0] < ans[0]:
    ans = ans[0]+str(use[0])*(len(ans)-1)

if N-int(ans) > 0:
    ans = str(use[0]) + str(use[0])*len(str(N)) if use[0]!=0 else str(use[1])+str(use[0])*len(str(N))



print(ans)

